.github/workflows/build-php.yaml

```diff
@@ -38,6 +38,16 @@ jobs:
             flavor: -slim
             build-php-cli: false
             version: 8.2.0
+          - name: php-
+            suffix: ""
+            flavor: ""
+            build-php-cli: false
+            version: 8.2.6
```

Comment:
Are we keeping both 8.2.0 and 8.2.6?

Reply:
For now. Will remove 8.2.6 once we have everything building for 8.2.6 and it is tested with Drupal and WordPress.